cdn/spec/resources/key.txt

```diff
@@ -0,0 +1 @@
+nZtRohdNF9m3cKM24IcK4w==
```

Comment:
you can probably just hardcode this string, and make `signed_url` accept this string, instead of accepting a file. this way, the sample code wouldn't require a "file" and users of the snippet can just pass their in-memory strings instead of writing them to a file first.

Reply:
Thanks @ahmetb, do you mean the runner should accept the key as a parameter and not a file?